[CIT-417] Address mqtt port change issues (#303)
thin-edge/thin-edge.io: docs/src/howto-guides/008_config_local_mqtt_port.md

```
This prints out the port that has been set.

## Step 3: Connect the thin edge device to cloud

Using `tedge` command connect to the desired cloud as below.

> This will force all the services (mosquitto, tedge_mappers) to use newer port.

Reviewer: ```suggestion
This will force all the services (mosquitto, tedge_mappers) to use newly set port.
```

Author: fixed

```shell

> tedge connect c8y/az

Reviewer: This line is a little bit confusing, I would rather you use specific command intended because when someone copy paste this one it will fail.
```suggestion
tedge connect c8y
# or
tedge connect az
```

Author: fixed

```

Note: The step 1 and 2 can be followed in any order.

## Update to use default port

To use the default port (1883), the mqtt.port has to be unset using the `tedge` command as below

```shell
tedge config unset mqtt.port
```

> Once the port is reverted to default, the step 1 and 3 has to be followed to use the default port.

Reviewer: Maybe you could provide links to sections with step 1 and step 3, there was one above where link to section could be useful. 
```suggestion
Once the port is reverted to default, the [step 1](#Step-3:-Connect-the-thin-edge-device-to-cloud) and 3 has to be followed to use the default port.
```

Author: fixed

```shell
tedge config set mqtt.port '"1024"'

> Error: failed to set the configuration key: mqtt.port with value: "1024".

Reviewer 2: Not sure if it is a good idea to use 1024 as this is the first  (or last) that can be used without elevated privledges

Author: ok changed it to 1234

Caused by:
    Conversion from String failed
```

## Updating the mqtt port in collectd

> Update the `collectd.conf` with the newer port in `<Plugin mqtt>`

Reviewer: ```suggestion
Update the `collectd.conf` with the new port in `<Plugin mqtt>`
```

What is `<Plugin mqtt>`?

Author: Thats a mqtt plugin in collectd.conf
